ocs_ci/deployment/hub_spoke.py

```
@@ -3980,7 +3980,7 @@ def setup_hypershift_oidc(self, bucket_name=None):
                 bucket_name = configured_bucket_name
                 logger.info(f"Using configured OIDC bucket name: {bucket_name}")
             else:
-                bucket_name = f"{self.name}-oidc-bucket"
+                bucket_name = f"{self.name}-oidc-{self.aws_region}"
                 logger.info(f"Using default OIDC bucket naming: {bucket_name}")
```

Ruff (0.16.0)

[warning] Logging statement uses f-string

(G004)